src/Microsoft.AspNetCore.Mvc.Formatters.Xml/SerializableErrorWrapper.cs

```diff
@@ -14,6 +14,10 @@ namespace Microsoft.AspNetCore.Mvc.Formatters.Xml
     [XmlRoot("Error")]
     public sealed class SerializableErrorWrapper : IXmlSerializable, IUnwrappable
     {
+        // Silly spelling of MVC-Empty to avoid collisions with other ModelState entries and emphasize element's unique
+        // nature.
+        private static readonly string EmptyKey = "MV\x00C7-\x00C9m\x00FEt\x00FF";
```

Comment:
@Eilon \ @mkArtakMSFT suggestions for this? We're trying to special case an empty model state key. This string is something user code could interact with

Reply:
@Eilon :ok: how 'bout `<MVC-Empty>` or perhaps `<MVC-ModelState-EmptyKey>`? I prefer the shorter name but leave it up to you.

FYI the upper Latin-1 was a holdover I should have removed when adding the illegal-in-C# dash.

I'll continue to use the same `EmptyKey` value in both `SerializableErrorWrapper` and `HttpError`.

Reply:
I don't think it shouldn't say MVC at all. We don't want to leak MVC-isms out into wire formats. This is ultimately not an MVC error, it's a validation error that we're letting the client know about.

Reply:
And, oops, it was @NTaylorMullen who suggested making it clear the element relates to MVC.

Reply:
Would it make sense to structure the responses better so that we can avoid bizarre keys?

For example:
1. Keep everything *by default* exactly as it was, aside from don't throw exceptions where we would currently throw exceptions
2. In CompatMode = 2.2, use a new serialization format that makes more sense, such as something like:
```xml
<Error>
    <FormLevel msg="The blah isn't blah" />
    <PropertyLevel key="prop1.Foo" msg="Prop1 is blah" />
    <PropertyLevel key="prop2.Bar" msg="Prop2 is blah" />
</Error>
```

And we can avoid all these shenanigans?

Reply:
Let's talk offline. I'm unsure about complicating the de-serialization side of this.

Reply:
Can you start an email thread and we can meet if necessary?